pulse2percept/utils/geometry.py

```diff
@@ -221,24 +284,106 @@ def plot(self, transform=None, label=None, style='hull', autoscale=True,
 
 
 class VisualFieldMap(object, metaclass=ABCMeta):
-    """Base class for a visual field map (retinotopy)
+    """ Base template class for a visual field map (retinotopy) """
 
-    A template
+    @abstractmethod
+    def region_mappings(self):
```

Comment:
`region_mappings` and `inv_region_mappings` is a bit confusing to a novice, because the direction of the mapping is not clear. Maybe make the name more explicit, e.g. `maps_from_dva`